src/app.py

```diff
@@ -120,8 +124,12 @@ def handle_file(experiment_id, f):
         update_dynamoDB(f)
         logger.info(f"\tMocked {filename} loaded into local DynamoDB.")
 
-    elif re.match("^r.rds.gz$", filename):
-        update_S3_count_matrix(experiment_id, f)
+    elif re.match("^biomage-source.r.rds.gz$", filename):
+        update_S3_count_matrix(experiment_id, f, SOURCE_BUCKET_NAME)
+        logger.info("\tMocked experiment data successfully uploaded to S3.")
```

Comment:
[suggestion, feel free to skip] To make this log different from the next one on like 133 and easier to read, you can add the name of the bucket experiment data was downloaded from in the log.

Reply:
changed